libraries/chain/include/eosio/chain/hotstuff.hpp

```diff
@@ -78,9 +119,16 @@ namespace eosio::chain {
       }
    };
 
+   
+   using hs_proposal_message_ptr = std::shared_ptr<hs_proposal_message>;
+   using hs_vote_message_ptr = std::shared_ptr<hs_vote_message>;
+   using hs_new_view_message_ptr = std::shared_ptr<hs_new_view_message>;
+   using hs_new_block_message_ptr = std::shared_ptr<hs_new_block_message>;
+
 } //eosio::chain
 
-// // @ignore quorum_met
+
+FC_REFLECT(eosio::chain::view_number, (_block_height)(_phase_counter));
```

Comment:
None of the other types in this file use `_` prefix on member data. Should remove these to be consistent.